common/metrics/defs.go

```diff
@@ -1412,6 +1357,8 @@ const (
 	WorkflowCronBackoffTimerCount
 	WorkflowCleanupDeleteCount
 	WorkflowCleanupArchiveCount
+	WorkflowCleanupSendArchiveSignalFailedCount
```

Comment:
There's an old metrics called ArchiverClientSendSignalFailureCount which is not used anywhere. I moved it here.